static/app/views/dashboards/widgets/common/widgetFrame.stories.tsx

```diff
@@ -71,6 +71,33 @@ export default storyBook(WidgetFrame, story => {
     );
   });
 
+  story('Badge', () => {
+    return (
+      <Fragment>
+        <p>
+          <JSXNode name="WidgetFrame" /> supports a <code>badge</code> prop. If passed, a
+          <code>Badge</code> component with the relevant props appears in the header.
+        </p>
+
+        <SideBySide>
+          <NormalWidget>
+            <WidgetFrame
+              title="count()"
+              badge={{
```

Comment:
Would the API be clearer if we called it `badgeProps` instead of `badge`?

Reply:
I like this idea! Changed